esti/lakectl_util.go

```diff
@@ -125,6 +125,7 @@ func sanitize(output string, vars map[string]string) string {
 	s = normalizeCommitID(s)
 	s = normalizeChecksum(s)
 	s = normalizeShortCommitID(s)
+	s = normalizeEndpoint(s)
```

Comment:
What changed that requires this extra sanitation?

Reply:
lakeCtl Doctor test that check for wrong endpoint expects 'http://lakefs:8000/api/v11' in the URL string. We don't want to rely on the DNS (when we run it local its localhost) - we need to sanitize it.